javascript-functions-practice/get-first-character.js

```diff
@@ -0,0 +1,9 @@
+/* exported getFirstCharacter */
+
+const voldemort = 'He who shall not be named';
+
+function getFirstCharacter(string) {
+  const firstChar = string['0'];
```

Comment:
Just do `string[0]`. Strings get converted to numbers so it works, but we don't want to rely on that, and for clarity sake, we should just use numbers.